pkgs/development/compilers/flutter/default.nix

```diff
@@ -6,26 +6,28 @@ let
     let files = builtins.attrNames (builtins.readDir dir);
     in map (f: dir + ("/" + f)) files;
 in {
-  stable = mkFlutter {
+  stable = mkFlutter rec {
     pname = "flutter";
     channel = "stable";
-    version = "1.12.13+hotfix.9";
-    sha256Hash = "1ql3zvmmk5zk47y30lajxaam04q6vr373dayq15jv4vpc0fzif1y";
+    version = "1.17.3";
+    filename = "flutter_linux_${version}-${channel}.tar.xz";
```

Comment:
It seems like flutter changed the naming scheme everywhere, this should probably be changed in `flutter.nix` to prevent boilerplate code.